api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -79,6 +79,11 @@ const (
 	ClusterInputErrorMsg      string = "The Spec could not be synced due to an input error:"
 )
 
+const (
+	CpuArchitectureX86 string = "x86_64"
```

Comment:
can we have those consts in common and use them for cloud and ztp?

Reply:
already moved to common in the next upload